package.json

```diff
@@ -18,22 +18,14 @@
   "author": "Suraj Tiwari <surajtiwari020@gmail.com>",
   "license": "MIT",
   "peerDependencies": {
-    "react": "*",
-    "react-native": "*"
+    "react-native": "^0.57.4"
   },
   "dependencies": {
     "fbemitter": "^2.1.1"
   },
   "devDependencies": {
-    "@callstack/eslint-config": "^2.0.0",
     "@types/fbemitter": "^2.0.32",
     "@types/react-native": "^0.57.4",
-    "eslint": "^5.6.1",
-    "eslint-plugin-react-native": "^3.3.0",
-    "flow-bin": "~0.67.1",
-    "react": "16.3.1",
```

Comment:
Is it correct? We import React and React Native in many files and it's going to fail in dev mode when we remove it from devDependencies.

Reply:
Please reinstall node_modules locally and try to run example app. I can't see nor React-Native neither React in node_modules.

Reply:
```
cd example
rm -rf node_modules
yarn
file react-native
> react-native: directory
```

Seems like it's there :)

Reply:
Ok my bad, i was trying to run example app not from example directory :D